package.json

```diff
@@ -130,7 +134,7 @@
     "uuid": "2.0.1",
     "valid-url": "1.0.9",
     "walk": "2.3.4",
-    "webpack": "1.13.1",
+    "webpack": "2.1.0-beta.25",
```

Comment:
It seems to work, but I didn't test it carefully. We should probably upgrade even if we don't decide to use this approach.